servers/alzirr/deployment.nix

```diff
@@ -0,0 +1,109 @@
+{ config, pkgs, lib, options, inputs, ... }:
+let
+  swampwalk-profile = "/nix/var/nix/profiles/per-user/buildkite-swampwalk/swampwalk";
+  swampwalk-frontend-profile = "/nix/var/nix/profiles/per-user/buildkite-swampwalk/swampwalk-frontend";
+
+in
+{
+  imports = [
+    ../.
+  ];
+
+  vault-secrets = {
+    vaultAddress = "https://vault.serokell.org:8200";
+    vaultPathPrefix = "kv/sys";
+    namespace = "tt";
+  };
+
+  nix.nixPath = options.nix.nixPath.default ++ [ "nixpkgs-overlays=/etc/nix/overlays.nix" ];
+  environment.etc."nix/overlays.nix".source = ./overlays.nix;
+  nixpkgs.overlays = import ./overlays.nix;
+
+  environment.systemPackages = with pkgs; [
+    stack
+    git
+    htop
+    nnn
+    vim
+    rsync
+    tmux
+    python
+    rebar3
+    elixir
+    erlang
+    cargo
+    gcc
+  ];
+
+  systemd.services.swampwalk = {
+    wantedBy = [ "multi-user.target" ];
+    environment.TODO_SWAMP_1_BASE_PATH = "/home/share";
+    environment.NIX_PATH = builtins.concatStringsSep ":" (options.nix.nixPath.default ++ [ "nixpkgs-overlays=/etc/nix/overlays.nix" ]);
+    path = [ "/run/wrappers" ];
+    serviceConfig = {
+      Restart = "on-failure";
+      User = "sweater";
+      Group = "users";
+      ExecStart = "${swampwalk-profile}/bin/swampwalk-server";
+    };
+  };
+
+  # add buildkite-swampwalk user to push and apply service updates from swampwalk pipeline
+  serokell-users.regularUsers = [ "buildkite-swampwalk" ];
```

Comment:
```
  users.users.deploy = {
    useDefaultShell = true;

    openssh.authorizedKeys.keys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIA1MvqWKMAejgaBfm0mXqwRK7QZ6NNOzCGj9aX+tiiow" ];
  };
```
? (this is how it's done on sadalbari, and yes I updated this to be the right ssh key already)